pkg/kubeapiserver/authorizer/config.go

```diff
@@ -82,7 +82,7 @@ func (config AuthorizationConfig) New() (authorizer.Authorizer, authorizer.RuleR
 				graph,
 				config.InformerFactory.Core().InternalVersion().Pods(),
 				config.InformerFactory.Core().InternalVersion().PersistentVolumes(),
-				config.VersionedInformerFactory.Storage().V1alpha1().VolumeAttachments(),
+				config.VersionedInformerFactory.Storage().V1beta1().VolumeAttachments(),
```

Comment:
NodeAuthorizer should handle BOTH `V1alpha1().VolumeAttachments` and `V1beta1().VolumeAttachments` not just one or the other.

Reply:
Since we can get both versions VolumeAttachment objects by V1beta1 or V1alpha1 API, so any API version here will do ?  Not very sure.  cc @liggitt

Reply:
No, it will only handle one. The authorizer runs in process in the apiserver, so it is never version skewed. I'd expect it to use v1beta1

Reply:
Thanks for the clarification, disregard this comment @NickrenREN.

Reply:
ok